### ISSUE TITLE
Mod Reader Info file bug

### ISSUE THREAD

**Issue by JavidPack** [blushiemagic/tModLoader#414]
The Info file retrieved from Mod Reader is a bit useless. You can read it with notepad and retrieve some text in an attempt to reconstruct build.txt. It might be useful to reconstruct build.txt or come up with a better approach.

The real issue, however, is if the Info file is in Mod Sources after attempting to retrieve source files from a .tmod file using the Extract option, the Info file seems to corrupt the .tmod file. The Mod menu will report new information like an updated version, but attempting to publish the .tmod file will result in the Mod Browser rejecting the mod because the version number is unchanged because the version reported to the Mod Browser mysteriously retrieved from that old Info file.

This is likely fixable by either updating the extract to convert Info to build.txt, updating building mods to ignore Info file since it would conflict with the automatic Info file, or updating building to catch the rare situation where 2 files with the same name are added via AddFile.

**Comment by Jofairden** [comment]
Since Info contains the build.txt information, I think converting Info to build.txt on extract makes sense. (input=build.txt output=build.txt)

**Comment by Chicken-Bones** [comment]
yeah unfortunately Info contains a little more than just the build information, for better or worse. For now I say we just add a reverse converter
